components/k8s-model-server/http-proxy/README.md

```diff
@@ -5,7 +5,7 @@ This component provides web server that proxy http rest api to Tensorflow/Servin
 
 The REST Api Spec is similar to Google Cloud Machine Learning Engine Prediction API.
 
-# API
+# API(rpc proxy)
```

Comment:
Here's a suggestion for how to maybe layout this document

- Explain why you would want to use HTTP proxy; i.e. that TF serving only supports gRPC so if you want to use HTTP we need a proxy that converts HTTP to gRPC

- Explain the model/TF serving requirements that must be satisfied in order to use HTTP Proxy
  (I think the model must have one of the supported signature methods predict or classify)

- Explain how to send requests;
   What the different methods are for and why to use one or the other.
   Explain the structure of the inputs/outputs
   Explain how to deal with binary data (e.g. b64 data).